### PR TITLE
[DEVOPS-497]  Buildkite CI

### PR DESCRIPTION
**Pull request by deepfire**
This adds integration with https://buildkite.com/input-output-hk.

### CODE REVIEW

**Comment by gromakovsky** [comment]
Please make someone from DevOps owner of that file/directory :)

**Comment by deepfire** [comment]
@gromakovsky, done!

**Comment by domenkozar** [comment]
Let's rather rename that to `@input-output-hk/devops`

File: CODEOWNERS [hunk 1]
```diff
@@ -23,6 +23,7 @@ appveyor.yml                    @domenkozar
 nixpkgs-src.json                @domenkozar
 pkgs/generate.sh                @domenkozar
 pkgs/default.nix                @ignore-autogenerated-file
+scripts/ci/                     @domenkozar
```
Comment (domenkozar) [inline CODEOWNERS]: should be @iohk-devops team

File: scripts/ci/ci.sh [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+#projects="core db lrc infra update ssc godtossing txp"
+#to_build=''
+
+#for prj in $projects; do
+#  to_build="$to_build cardano-sl-$prj"
+#done
+
+for trgt in $targets; do
+  echo "Prebuilding dependencies for $trgt, quietly.."
+  nix-shell -A $trgt --run true --no-build-output --cores 0 --max-jobs 4 default.nix ||
```
Comment (domenkozar) [inline scripts/ci/ci.sh]: This rebuilds targets now twice, we should only build verbosely.

Also we should add `cardano-sl-auxx` to release.nix to prebuilt deps.

Comment (deepfire) [reply scripts/ci/ci.sh]: Ok, will do.  (The second rebuild only happens for a failing dependency, though.)

**Pull request merged**